### DISCUSSION TITLE
HTTP error 429

### DISCUSSION THREAD

**Discussion by Beratf2** [TF2Autobot/tf2autobot#1698] [Q&A]
2024-01-20 22:12:35 error: TF2Autobot failed to start properly, this is most lik
ely a temporary error. See the log:
package.version: 5.9.1; node: v20.11.0 win32 x64}
Stack trace:
Error: HTTP error 429

help guys

**Comment by excellproj** [comment]
Create a Steam API key( https://steamcommunity.com/dev/apikey ,  log in with your bot's credentials) and set `localhost` as the website(Domain Name).
After creating the key, wait for 15 minutes and try to start the bot

**Reply by Beratf2** [reply]
Where do I write the steam api key?

**Reply by Beratf2** [reply]
![Ekran görüntüsü 2024-01-21 161511](https://github.com/TF2Autobot/tf2autobot/assets/157242918/4551ae64-2d71-4709-b974-bb50554f7dbc)

**Reply by excellproj** [reply]
This key needs to be generated for the bot, and that's it(**It's important to note that logging in should be done by the bot itself, not by another account.**). You don't need to write it. Another question: do you have a full version of the Steam account?
https://github.com/TF2Autobot/tf2autobot/wiki/Before-you-start#2-upgrade-your-new-steam-account-to-premium-status-in-team-fortress-2

**Reply by Beratf2** [reply]
yes my steam account is premium

**Reply by excellproj** [reply]
In that case, everything should work fine.

**Reply by Beratf2** [reply]
can you help me on discord ?

**Reply by excellproj** [reply]
https://discord.gg/4k5tmMkXjB
